README.md

```diff
@@ -60,6 +60,19 @@ Twig components can also be generated by changing directory into the project:
 yo hozokit:component
 ```
 
+#### Further Steps
+
+Once the generator has finished running the following steps might be required if not done already:
+
+1. Setup a webserver cabable of running php and create a mySQL database for Wordpress. Read [how to install Wordpress](https://wordpress.org/support/article/how-to-install-wordpress/) to learn more.

```

Comment:
Change 'cabable' to 'capable'.
Change 'php' to 'PHP'.
Change 'wordpress' to 'Wordpress'.

Reply:
Changed! 👌